ably/auth.go

```diff
@@ -238,7 +260,7 @@ func (a *Auth) requestToken(params *TokenParams, opts *AuthOptions) (tok *TokenD
 // Auth.Authorize instead.
 //
 // Refers to RSA10l
-func (a *Auth) Authorise(params *TokenParams, opts *AuthOptions) (*TokenDetails, error) {
+func (a *Auth) Authorise(params *TokenParams, opts *authOptions) (*TokenDetails, error) {
```

Comment:
This is a public method, I believe public methods should receive public arguments. I'm wondering why opts it private ?

Reply:
This was an oversight, thanks. Fixed at e51a85a.